lib/puppet.rb

```diff
@@ -33,6 +33,10 @@
   def _(msg)
     msg
   end
+
+  def n_(msg_singular, msg_plural, count)
+    return count == 1 ? msg_singular : msg_plural
```

Comment:
So, actually, I'm unsure what you mean about indexing on count. Count is the bit of the sentence that determines which form to use based on linguistic rules, and is unrelated to how many args there are. In English, there are only two plural forms, ever, which means this method will only ever be called with two args plus the count as long as we're comfortable assuming it will only be called in an English context. The gettext version supports more because they need to be flexible enough to support other "native" languages which may have additional plural forms.